### ISSUE TITLE
During the training phase, the gradient will gradually become large

### ISSUE THREAD

**Issue by TfeiSong** [yuantianyuan01/StreamMapNet#15]
During the training phase, the gradient will gradually become large, do you know what causes this ？ 
Looking forward to your reply

![1701676667228](https://github.com/yuantianyuan01/StreamMapNet/assets/25846975/742c83b1-e8bb-421c-9399-96cf02e24348)

**Comment by yuantianyuan01** [comment]
What batch size do you use? It seems failed to converge at all and I suspect it's caused by very small batch size. Can you share your training config file?

**Comment by TfeiSong** [comment]
[https://github.com/yuantianyuan01/StreamMapNet/blob/main/plugin/configs/nusc_newsplit_480_60x30_24e.py](url)
Almost the same as above,  but only one gpu is used and total train sample numbers is 19291。

**Comment by yuantianyuan01** [comment]
So your real batch size is 1/8 compared with using 8 GPUs and it may cause instability. You can try using lower learning rates or larger batch size to solve the problem.

**Comment by TfeiSong** [comment]
Thank you very much. I'll try right away
